api/models/resource_request.go

```diff
@@ -31,6 +31,12 @@ type ResourceRequest struct {
 	CPURequest resource.Quantity `json:"cpu_request"`
 	// Memory request of inference service
 	MemoryRequest resource.Quantity `json:"memory_request"`
+	// GPU resource type (nvidia.com/gpu or amd.com/gpu)
```

Comment:
nit: I think it's better use `GPU` instead of `Gpu` to make it consistent with `CPURequest` above. Ref: [https://github.com/golang/go/wiki/CodeReviewComments#initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)